spec/acceptance/lastEvaluatedKeyWithFilterAndPagingTest.spec.ts

```diff
@@ -257,19 +260,18 @@ describe('lastEvaluatedKeyWithFilterAndPagingTest', function ()
        assert.isUndefined(page2.lastEvaluatedKey);
 
        let page1Again = await nodenamo.list().from(User).filter(filter).limit(3,3).order(false).resume(page2.firstEvaluatedKey).execute<User>();
-       assert.deepEqual(page1Again.items, [
-           user7,
-           user6,
-           user2
-       ]);
+       assert.deepEqual(page1Again.items[0], user7);
+       assert.deepEqual(page1Again.items[1], user6);
+       assert.deepEqual(page1Again.items[2], user2);
+       assert.equal(page1Again.items.length, 3);
        assert.isDefined(page1Again.firstEvaluatedKey);
        assert.isDefined(page1Again.lastEvaluatedKey);
 
        let page2Again = await nodenamo.list().from(User).filter(filter).limit(3,3).resume(page1Again.firstEvaluatedKey).execute<User>();
-       assert.deepEqual(page2Again.items, [
-           user8
-       ]);
+       assert.deepEqual(page2.items[0], user8);
+       assert.equal(page2.items.length, 1);
        assert.isDefined(page2Again.firstEvaluatedKey);
+       assert.equal(page2.firstEvaluatedKey, page2Again.firstEvaluatedKey);
        assert.isUndefined(page2Again.lastEvaluatedKey);
```

Comment:
Can we add assert.deepEqual(page2Again.lastEvaluatedKey, page2.lastEvaluatedKey) ?